xc/xc7/toolchain_wrappers/symbiflow_synth

```diff
@@ -5,13 +5,9 @@ MYPATH=`realpath $0`
 MYPATH=`dirname ${MYPATH}`
 
 export SHARE_DIR_PATH=`realpath ${MYPATH}/../share/symbiflow`
-export TECHMAP_PATH=${SHARE_DIR_PATH}/techmaps/xc7_vpr/techmap
```

Comment:
The `TECHMAP_PATH` may change based on where the synth is executed. For instance, in the `symbiflow-arch-defs` tests, the techmap path corresponds to this `https://github.com/SymbiFlow/symbiflow-arch-defs/tree/master/xc/xc7/techmap`, while in the installed toolchain, the techmap PATH is in a different location as specified in here: https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/xc/common/cmake/install.cmake#L74.

I am not quite sure how to handle different paths locations without having the `TECHMAP_PATH` variable

Reply:
I see. Let's leave TECHMAP_PATH in there then. I'll fix